src/loader/loader.c

```diff
@@ -113,6 +118,10 @@ static bool guc_disable_load = false;
 static post_parse_analyze_hook_type prev_post_parse_analyze_hook;
 static shmem_startup_hook_type prev_shmem_startup_hook;
 
+#if PG_VERSION_SUPPORTS_MULTINODE
```

Comment:
I guess we might want to come up to consistent set of macroses for version checking, can be reworked when we remove support for older versions

Reply:
I guess this will go away when we remove support for older PG versions.